.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @sfc-gh-japatel @sfc-gh-rramachandran @sfc-gh-mnaides @sfc-gh-tzhang @sfc-gh-tjones
+* @sfc-gh-japatel @sfc-gh-tzhang @sfc-gh-tjones @sfc-gh-rcheng
```

Comment:
I think this won't work long term, we should just add Snowpipe and Snowpipe Streaming team, and whoever wants to be added explicitly could do it themselves, WDYT?

Reply:
Sounds like a plan! I will do it.